### PR TITLE
👩‍🚀 Prism: Interactive Aurora Refractor Laboratory Page

### PR DESCRIPTION
**Pull request by alehar9320**
Autonomous creative update from Prism 👩‍🚀 introducing an interactive "Aurora Refractor" HTML5 canvas element inside the laboratory directory (`src/pages/lab/aurora-refractor.astro`) and directory listing (`src/pages/lab/index.astro`). Powered by pure math utilities for sine wave superposition and magnetosphere vector displacement. Includes accessibility controls, `prefers-reduced-motion` fallbacks, keyboard shortcuts, and `astro:before-swap` cleanup.

---
*PR created automatically by Jules for task [14964668392165734510](https://jules.google.com/task/14964668392165734510) started by @alehar9320*

## Summary by CodeRabbit

* **New Features**
  * Added an Interactive Experiments laboratory with an Aurora Refractor canvas experience.
  * Added animated aurora waves with palette controls and pause/play functionality.
  * Added mouse and touch interactions for shaping the magnetosphere.
  * Added keyboard shortcuts, responsive layouts, adaptive rendering, and reduced-motion support.
  * Added navigation from the laboratory index to the Aurora Refractor experiment.

* **Tests**
  * Added coverage for aurora animation calculations, palette handling, interpolation, and edge cases.

### CODE REVIEW

**Comment by alehar9320** [comment]
Not merging. Jules under a Prism title (jules.google.com task, `.jules/prism.md`, aurora refractor lab page). MERGEABLE, not draft, not in the keep list. Does not belong on the hire site. No Jules onto main. Overlay 69ca717 and #597 stay untouched.

**Review by alehar9320 — COMMENTED** [review]
Copy-only review.

This adds a public `/lab/` with share titles and page copy. The canvas controls (Pause, Palette, keyboard hints) are fine. The rest reads like a physics writeup, not a person: “high-impact visual experiments,” “mathematical wave superposition,” “magnetosphere vector forces.” Same corporate/bot voice we have been taking off the hire pages.

Alexander already said this does not belong on the hire site. From writing, I agree: do not ship this copy, and do not treat a rewrite as a path to merge. If a lab page ever does ship, the visitor lines should be what you see and what to do, not how the math works.

Also: `/experimental/now` is `noindex`. These lab pages are not. If they ever go public, this copy is the Google/LinkedIn preview.

File: src/pages/lab/index.astro [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  ogTitle="Laboratory & Experiments"
+>
+  <main id="main-content" class="wrapper stack lab-index">
+    <header class="hero">
+      <p class="eyebrow">Laboratory & Sandbox</p>
+      <h1 class="title">Interactive Experiments</h1>
+      <p class="subtitle">
+        A dedicated laboratory space for high-impact visual experiments, mathematical art, and
+        interactive canvas prototypes.
+      </p>
```
Comment (alehar9320) [inline src/pages/lab/index.astro:17]: Hero + share copy is brochure-bot. “High-impact” is the same filler we killed on the contact CTA. “Laboratory showcasing creative interactive web art, canvas simulations, and digital experiences” is a keyword pile, not a sentence a visitor needs.

If this index exists at all:

- title: `Lab | Alexander Härenstam`
- description: `Small interactive pieces. Not case studies.`
- ogTitle: `Lab`
- eyebrow: `Lab`
- h1: `Experiments`
- subtitle: `Small interactive pieces. Not case studies.`

File: src/pages/lab/index.astro [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    <section class="experiments-grid" aria-label="Laboratory Experiments">
+      <a class="experiment-card" href="/lab/aurora-refractor/">
+        <div class="card-tag">HTML5 Canvas &bull; Math Superposition</div>
+        <h2 class="card-title">Aurora Refractor</h2>
+        <p class="card-desc">
+          Interactive Northern Lights simulation built with multi-layered sine wave superposition,
+          magnetosphere vector forces, and customizable color refractions.
+        </p>
+        <span class="card-link">Explore Experiment &rarr;</span>
+      </a>
```
Comment (alehar9320) [inline src/pages/lab/index.astro:29]: The card talks like an implementation note. “HTML5 Canvas • Math Superposition” and “magnetosphere vector forces” belong in a commit, not on the page.

Try:

- tag: `Northern lights`
- title: `Aurora Refractor` (this one can stay)
- desc: `Move around the canvas. The lights follow.`
- link: `Try it →`

File: src/pages/lab/aurora-refractor.astro [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+  ogTitle="Aurora Refractor Canvas Lab"
+>
+  <main id="main-content" class="wrapper stack lab-page">
+    <header class="lab-header">
+      <p class="lab-badge">Laboratory & Experiment</p>
+      <h1 class="lab-title">Aurora Refractor</h1>
+      <p class="lab-subtitle">
+        A mathematical wave superposition canvas driven by magnetosphere vector physics and Northern
+        Lights refractions.
+      </p>
```
Comment (alehar9320) [inline src/pages/lab/aurora-refractor.astro:17]: This description is the share preview. “Canvas refractor lab with mathematical wave superposition and magnetosphere interaction” will not read as Alexander.

If this page exists at all:

- title: `Aurora Refractor | Alexander Härenstam`
- description: `Northern lights on a canvas. Move your pointer and they bend.`
- ogTitle: `Aurora Refractor`
- badge: `Lab`
- subtitle: `Northern lights on a canvas. Move your pointer and they bend.`

Pause / Palette / keyboard hints can stay.
